.travis.yml

```diff
@@ -1,39 +1,36 @@
 dist: xenial
 language: python
+
 matrix:
   include:
   - python: "2.7"
-    env: TORNADO_VERSION=4.4.3 NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2
-  - python: "2.7"
-    env: TORNADO_VERSION=4.5.3 NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2
+    env: TORNADO_VERSION=4.1   NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
   - python: "2.7"
     env: TORNADO_VERSION=4.4.3 NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
-  - python: "2.7"
-    env: TORNADO_VERSION=4.5.3 NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
   - python: "2.7"
     env: TORNADO_VERSION=4.5.3 NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3
+  - python: "2.7"
+    env: TORNADO_VERSION=5.1.1 NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3
 
-  - python: "3.4"
-    env: TORNADO_VERSION=4.4.3 NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2
   - python: "3.4"
     env: TORNADO_VERSION=4.5.3 NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
+  - python: "3.4"
+    env: TORNADO_VERSION=5.1.1 NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3
 
-  - python: "3.5"
-    env: TORNADO_VERSION=4.5.3 NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2
-  - python: "3.5"
-    env: TORNADO_VERSION=4.4.3 NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
   - python: "3.5"
     env: TORNADO_VERSION=4.4.3 NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3
+  - python: "3.5"
+    env: TORNADO_VERSION=5.1.1 NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8
```

Comment:
should be nsq `1.1.0`

Reply:
sure, but, fwiw, I was trying to mix it up so so that e.g. 5.1.1 was tested with nsqd-1.0.0 in some case and 4.5.3 was tested with nsqd-1.1.0 in some case (without expanding the test matrix too much)